profiles/base.yaml

```diff
@@ -193,6 +193,7 @@ crossing_training_requires_movement:
 - Trading
 forage_item:
 braid_item:
+collect_item: rock
```

Comment:
I was thinking about updating crossing-trainer to use this change, too, especially since people seem to be congregating now in the empath guild where rocks are not available, but didn't think it fit in the same commit.

The setting is inherited by the script that call it, so in the current implementation it's possible to override the collect_item in each yaml, letting you have a different collect_item in your crossing_training, primary hunt and secondary hunts.